nanopub/publication.py

```diff
@@ -130,12 +130,19 @@ def from_assertion(cls, assertion_rdf, uri=DEFAULT_NANOPUB_URI, introduces_conce
                             attributed_to))
 
         if derived_from:
-            # Convert derived_from URI to an rdflib term first (if necessary)
-            derived_from = rdflib.URIRef(derived_from)
-
-            provenance.add((this_np.assertion,
-                            namespaces.PROV.wasDerivedFrom,
-                            derived_from))
+            list_of_URIs = []
```

Comment:
```suggestion
            uris = []
```
Don't use capitals in variable names and it is is obvious that it is a list (since it's plural)

Reply:
And you actually don't have to to declare this variable because you are doing it in the if/else statements.

Reply:
OK, fair enough. I wanted to avoid confusion with the ```uri``` param, but that will hopefully be removed soon anyway.